src/ProcessManager.php

```diff
@@ -902,7 +902,7 @@ protected function newSlaveInstance($port)
         // we can not use -q since this disables basically all header support
         // but since this is necessary at least in Symfony we can not use it.
         // e.g. headers_sent() returns always true, although wrong.
-        $commandline = $this->phpCgiExecutable . ' -C ' . ProcessUtils::escapeArgument($file);
+        $commandline = $this->phpCgiExecutable . ' -C ' . self::escapeArgument($file);
```

Comment:
Is there a use we can remove, too? Does maybe mkraemer contain an escaping tool we could reuse instead of duplicating code? Otherwise fine with me :)

Reply:
Maybe, but I`m not sure. 
Duplicating code is very stable fix of the problem i think ;)